src/adapter/src/config/frontend.rs

```diff
@@ -151,17 +167,29 @@ impl SystemParameterFrontend {
 
 #[derive(Debug, Clone)]
 struct Metrics {
-    pub last_known_time_seconds: IntCounter,
+    // TODO: remove this in favor of last_cse_time_seconds.
+    pub last_known_time_seconds: UIntGauge,
```

Comment:
If this is changing types it'll mess up all the prom graphs anyway, so can it just be removed now?

Reply:
Soooo all Prometheus metrics are actually doubles under the hood. Not making this up. The types here are just for convenience; there's an `as f64` inside the Prometheus library. So the type here actually isn't changing as far as Prometheus is concerned.

Reply:
Assuming you're talking about the `TYPE` field in the /metrics endpoint, the values are "counter" or "gauge" or "histogram". No mention of e.g. int vs double, because everything is a double!

```
# HELP mz_persist_audit_blob_bytes total size of blob
# TYPE mz_persist_audit_blob_bytes gauge
mz_persist_audit_blob_bytes 0
```

Reply:
Ah, we're talking about different things. I meant the change from counter to gauge.

Reply:
OH. Good point.

Reply:
> But the reported metadata type will change, so the graphs will interpret all data with the new type?

I'll test if the `counter ⇒ gauge` change leads to notable changes in the graphs before merging this.

Reply:
I don't see a problem in the Prometheus expressions of our current panel definitions from using `gauge` instead of `counter`.